### PR TITLE
On windows, spawn needs to call node in order to execute js files.

### PR DESCRIPTION
**Pull request by TwitchBronBron**
The cli tests fail when running on windows. I think it has to do with a js file not being executable on its own. This PR fixes that.

### CODE REVIEW

File: test/cli/cli.test.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ describe("cli", () => {
         let rootDir = path.join(__dirname, "resources");
 
         return execFile(
```
Comment (sjbarag) [inline test/cli/cli.test.js]: We might want to switch these to a plain `exec()` if we're calling `node` as the command - looks like `execFile` expects the first argument to be the file to execute!

Comment (sjbarag) [inline test/cli/cli.test.js]:
```suggestion
         return exec(
```

File: test/cli/cli.test.js [hunk 2]
```diff
@@ -21,8 +21,11 @@ describe("cli", () => {
 
     it("defaults --root to process.cwd()", () => {
         return execFile(
```
Comment (sjbarag) [inline test/cli/cli.test.js]:
```suggestion
         return exec(
```

File: test/cli/cli.test.js (hunk 1 above)
Comment (sjbarag) [reply test/cli/cli.test.js]: I can't add suggestions to the `const execFile = ` line :cry:

File: test/cli/cli.test.js [hunk 3]
```diff
@@ -2,29 +2,34 @@ const child_process = require("child_process");
...
 
-const execFile = pify(child_process.execFile);
+const exec = pify(child_process.exec);
 
 describe("cli", () => {
     it("accepts a root directory", () => {
         let rootDir = path.join(__dirname, "resources");
 
-        return execFile(
+        let command = [
```
Comment (TwitchBronBron) [inline test/cli/cli.test.js]: exec takes a single string with the full command, so I abstracted the command out into its own line so it's more readable

**Comment by TwitchBronBron** [comment]
@sjbarag Ready for your review

**Pull request merged**